pkg/lib/cockpit-components-install-dialog.jsx

```diff
@@ -207,3 +218,56 @@
     check_missing();
     return prom;
 }
+
+function install_dialog_immutable(pkg, options) {
+    return new Promise((resolve, reject) => {
+        const missing_name = <strong>{pkg.join(", ")}</strong>;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18240-20230125-114047-de98707b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)